politeiawww/cmd/politeiavoter/politeiavoter.go

```diff
@@ -383,80 +377,10 @@ func (c *ctx) makeRequest(method, route string, b interface{}) ([]byte, error) {
 	return responseBody, nil
 }
 
-func (c *ctx) makeRequestFail(method, route string, b interface{}) ([]byte, error) {
```

Comment:
Why is this being taken out?  Dead code?

Reply:
yep